.evergreen/install-dependencies.sh

```diff
@@ -96,7 +97,11 @@ fi
 
 if [[ $operating_system != "win" ]]; then
   # Update npm to latest when we can
-  npm install --global npm@latest
+  if [[ $NODE_LTS_VERSION -lt $NODE_LOWEST_LTS ]]; then
```

Comment:
Note that our `run-benchmarks-node-18` task specifies `NODE_LTS_VERSION` as `v18.16.0` so this line results in a syntax error.

[Evergreen link](https://spruce.mongodb.com/task/node_bson_perf_run_benchmarks_node_18_patch_7b9840345a33a02a4d45a5afdb6da846abfd9570_64f59b370305b983ce7370a7_23_09_04_08_54_17/logs?execution=0)

Reply:
Updated to use just 18 and follow the same pattern as the other tasks that set this variable.

Reply:
Why not just add a `NODE_MAJOR_VERSION` variable up in L34-43 so that we can handle these things more generally for any other pinned versions down the line?